### DISCUSSION TITLE
Targeting .NET 6 iOS projects in nuspec

### DISCUSSION THREAD

**Discussion by frenzibyte** [dotnet/maui#12007] [Q&A] [answered]
With Xamarin, it's possible to provide specific DLL for iOS projects as [documented here](https://learn.microsoft.com/en-us/xamarin/cross-platform/app-fundamentals/nuget-manual). This is however not working on .NET 6 iOS (`net6.0-ios`). I've tried renaming `Xamarin.iOS` to `net6.0-ios` to no avail. Am I missing something?

**Comment by xtuzy** [comment, marked as answer]
you can create a default class library for net6.0-ios or maui, select generate nuget package when build, you can find a nuspec file in obj folder, it have all detail for net6.0-ios.
